test/test.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Tests to verify stub behaviors
+ */
+/// <reference types="@google/local-home-sdk" />
+import test from 'ava';
+import {
+  UDPDevice,
+  MockNetwork,
+  RemoteAddressInfo,
+} from '../platform/mock-radio';
+import {
+  MockLocalHomePlatform,
+  UDPScanConfig,
+  ScanState,
+} from '../platform/mock-local-home-platform';
+
+// Tests a UDP identify flow end-to-end
+test('udp-device-connects', (t) => {
```

Comment:
can you add test for the DeviceManager?

Reply:
Will add DeviceManager tests in another PR (not much to test right now)